app/assets/stylesheets/application.css

```diff
@@ -10,7 +10,10 @@ h2 {
 
```

Comment:
![image](https://github.com/dheerajsachdeva/blog-app/assets/93039359/3e875a8d-3aa4-46bc-a33b-093c3e825b98)

- [ ] It would be nice if you fix the link highlighted in the image above to make your readme descriptive and professional 👍